mobile/src/androidTest/java/ch/epfl/sdp/mobile/test/application/chess/notation/FlowsTest.kt

```diff
@@ -0,0 +1,37 @@
+package ch.epfl.sdp.mobile.test.application.chess.notation
+
+import ch.epfl.sdp.mobile.application.chess.engine.Delta
+import ch.epfl.sdp.mobile.application.chess.engine.Game
+import ch.epfl.sdp.mobile.application.chess.engine.Position
+import ch.epfl.sdp.mobile.application.chess.notation.mapToGame
+import ch.epfl.sdp.mobile.application.chess.notation.serialize
+import ch.epfl.sdp.mobile.test.application.chess.engine.play
+import com.google.common.truth.Truth.assertThat
+import kotlinx.coroutines.flow.flowOf
+import kotlinx.coroutines.flow.map
+import kotlinx.coroutines.flow.toList
+import kotlinx.coroutines.test.runTest
+import org.junit.Test
+
+class FlowsTest {
+
+  @Test
+  fun given_gameFlow_when_incrementalUpdate_then_endsInCorrectState() = runTest {
+    val game = Game.create().play { Position(0, 6) += Delta(0, -2) }
+    val one = game.serialize()
```

Comment:
Nitpick: what is game.serialize() -> is a game then maybe name it `gameOne`

Reply:
`serialize()` returns a `List<String>`